### ISSUE TITLE
Flux LoRA Training: Loss stuck at 1.9 and pixelated outputs (ComfyUI / FluxTrainer)

### ISSUE THREAD

**Issue by blitolyricalrain-coder** [kijai/ComfyUI-FluxTrainer#207]
Hi everyone,

i dont know where to post my problem, if anyone want to DM me its a pleasure. 
Its been a week and i'm desesperate.

I am trying to train my first LoRA on Flux in ComfyUI using the Kijai FluxTrainer nodes, but I am getting pixelated/noisy outputs and the training loss (avr_loss) is stuck around 1.9. I tried changing parameters multiple times without success.

My Setup:

OS: Windows 11
GPU: RTX 4070 12GB VRAM
Software: ComfyUI Portable (Latest version)
Model: Flux1-Dev (fp8)
Dataset: 23 images, all resized to 512x512.
with for each images a .txt starts with the trigger word.

The avr_loss usually stays very high (between 1.8 and 1.95).

History of attempts and issues:

Attempt 1: Trigger word marshall
Result: Loss started at 1.82 and dropped very slowly to 1.34, then got stuck. Outputs were pixelated.
Attempt 2 & 3: Trigger word apollowright (unknown token), Text Encoder enabled (lr=0.001), Min SNR Gamma = 1.00.
Result: Loss exploded to 1.9 or 1.91 from the start and stayed there. The model failed to learn.
Log note: The console showed text_encoder is not needed for training. deleting to save memory. Since the word is unknown and TE is disabled, I think it can't learn the token.
Attempt 4 (Current): Trigger word "man" (generic token), Text Encoder Disabled (lr=0), Min SNR Gamma = 1.00.
Result: same result, just fail

My Current Parameters: (you can see my screenshots)

Network Dim: 32
Network Alpha: 16
Learning Rate: 0.0004
Batch Size: 1 (Auto-lowered due to VRAM)
Min SNR Gamma: 1.00
Optimizer: CAME
Train Text Encoder: Disabled (clip l lr = 0, t5 lr = 0) -> Is this the problem?
Validation Prompts: Separated correctly with |.
Console Log (Current Start):

Epoch 1/10 - steps:   0%| ... | 1/2300 [00:12<7:40:38, 12.02s/it, avr_loss=0.49]
(One single step was 0.49, but it immediately went back to 1.9+ after a few steps).

Questions:

How resolve my problem and have a successful result ? thank you in advance.

<img width="3320" height="1072" alt="Image" src="https://github.com/user-attachments/assets/52ab2fdc-9dfd-425a-b968-d6edd7a83862" />
<img width="2859" height="1075" alt="Image" src="https://github.com/user-attachments/assets/2e3bca39-d59a-4b81-86be-6727e76f904d" />
<img width="3314" height="1173" alt="Image" src="https://github.com/user-attachments/assets/5598701e-6dc6-4e4b-bb94-93dbde8be53d" />
<img width="2905" height="1211" alt="Image" src="https://github.com/user-attachments/assets/aae3d363-9319-4b46-b6ed-5588deb7ff8b" />

**Comment by Floristort** [comment]
I'm running into the same issue, have not yet found a solution

**Comment by Floristort** [comment]
Found a solution, if you use the models from this reply, then it should work:

https://github.com/kijai/ComfyUI-FluxTrainer/issues/80#issuecomment-3670973747
